packages/valory/skills/safe_deployment_abci/rounds.py

```diff
@@ -71,47 +72,47 @@ class RandomnessSafeRound(CollectSameUntilThresholdRound):
     """A round for generating randomness"""
 
     allowed_tx_type = RandomnessPayload.transaction_type
-    payload_attribute = "randomness"
+    payload_attribute = get_name(RandomnessPayload.randomness)
     synchronized_data_class = SynchronizedData
     done_event = Event.DONE
     no_majority_event = Event.NO_MAJORITY
-    collection_key = "participant_to_randomness"
-    selection_key = "most_voted_randomness"
+    collection_key = get_name(SynchronizedData.participant_to_randomness)
+    selection_key = get_name(SynchronizedData.most_voted_randomness)
 
```

Comment:
A further benefit is that we can now more easily refactor the rounds and remove some of these class variables as they can be inferred automatically (e.g. `payload_attribute` could be inferred if we made the assumption that every transaction has exactly one payload attribute - which currently holds apart from one edge case that can be altered).